README.md

```diff
@@ -1,2 +0,0 @@
-# questionnaire
```

Comment:
Could you please also add some information about the required environment (php, mysql, composer etc) and on how to start the project (for example: 1) navigate to the project dir; 2) run compose install)